src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2684,13 +2684,13 @@ class AccessibilityHelpWidget extends Widget implements ITerminalWidget {
 	private readonly _markdownRenderer: MarkdownRenderer;
 
 	constructor(
-		instance: ITerminalInstance,
+		private readonly _instance: ITerminalInstance,
```

Comment:
I just noticed this is a circular dependency. Should we pass in `instance.xterm`, `instance.onDidFocus` instead and have an `onDidRequestFocus`?

Reply:
thought abt that but we also need `shellType`. probably should do it, but that felt like a lot to pass in

Reply:
👌 not ideal, but it's less of a problem since it's in the same file